pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/rbac/leader_election_role_binding.go

```diff
@@ -43,6 +44,11 @@ func (f *LeaderElectionRoleBinding) SetTemplateDefaults() error {
 const leaderElectionRoleBindingTemplate = `apiVersion: rbac.authorization.k8s.io/v1
 kind: RoleBinding
 metadata:
+  labels:
+    app.kubernetes.io/name: leader-election-rolebinding
+    app.kubernetes.io/component: leader-election
```

Comment:
```suggestion
    app.kubernetes.io/component: rbac
```

ihmo: The architecture component is RBAC or permissions  such as in the examples we have a database

Reply:
Should be finished in both `kustomize` plugins as of commit: https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/daaca98d6775fe54fda2a0017d2f0f4b1369c1de

Marking this particular comment as resolved.